e2e/src/app.e2e-spec.ts

```diff
@@ -40,7 +66,8 @@ describe('Band detail page', () => {
 
   it('should display band responsibilities when responsibilities panel is clicked', () => {
     page.clickResponsibilities();
-    expect(page.getResponsibilitiesContent()).toContain('Executive responsibility');
+    browser.driver.sleep(500);
```

Comment:
why do we need to do this?

Reply:
The animation for opening the expansion meant that the element wasn't immediately available